networking/solarxr/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "solarxr"
+version = "0.0.0"
+
+license.workspace = true
+authors.workspace = true
+repository.workspace = true
+
+edition.workspace = true
+rust-version.workspace = true
+
+
```

Comment:
not sure how rust devs handle this but normally in Python / JS land we just let prettier run over all our configs so we never get two empty lines

Reply:
why review useless stuff but not the actual PRs I request from ya smh

Reply:
This is because formatting of toml files is not done by rustfmt but by other tools like toml formatters. And the autoformatter for the toml files is not running in my neovim rn for some reason.

Reply:
idk, maybe because I dont wanna review rust code?